include/revng/Model/LoadModelPass.h

```diff
@@ -33,4 +35,6 @@ class LoadModelPass : public llvm::ImmutablePass {
     Modified = true;
     return TheBinary;
   }
+
+  static std::unique_ptr<model::Binary> getModel(const llvm::Module &M);
```

Comment:
Why all these `unique_ptr` business? If you're concerned about efficiency just use `std::move`.

Reply:
Done!